spec/requests/api/v1.1/root_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe "v1.1 - root", :type => [:request, :v1x1] do
+  it "#openapi.json" do
+    get("#{api_version}/openapi.json")
+
+    expect(response.content_type).to eq("application/json")
+    expect(response).to have_http_status(:ok)
+  end
```

Comment:
@mkanoor @eclarizio I believe the current v1.2 test is currently handled by https://github.com/RedHatInsights/catalog-api/blob/master/spec/requests/api/v1/root_spec.rb#L3, but when we eventually bump to the next version I would expect that to be changed as well.

Should we be created a v1.2 `root_spec` in the `v1.2` folder now so that we don't miss it on the next change?  If so, do wee need the one if here https://github.com/RedHatInsights/catalog-api/blob/master/spec/requests/api/v1/root_spec.rb#L3 or should that file just be the single `handles redirects correctly` spec?

Reply:
@gmcculloug Hmm, yeah, good question, I like the suggestion of moving the `v1/root_spec.rb` to a new `root_spec.rb` in the `v1.2` folder.

I think ultimately the `handles redirects correctly` spec probably either stays where it is and the `:type => []` array consistently gets changed to `:v1x3`, `:v1x4`, etc, or we continually move the whole `it` block forward into the various `/api/v1.2/root_spec.rb`, `/api/v1.3/root_spec.rb`, etc.